Formula/influxdb.rb

```diff
@@ -1,8 +1,8 @@
 class Influxdb < Formula
   desc "Time series, events, and metrics database"
   homepage "https://influxdata.com/time-series-platform/influxdb/"
-  url "https://github.com/influxdata/influxdb/archive/v1.8.3.tar.gz"
-  sha256 "d8b89e324ed7343c1397124ac3cc68c405406faf74e7369e733611cada54656d"
+  url "https://github.com/influxdata/influxdb/archive/v2.0.2.tar.gz"
```

Comment:
(Disclaimer: 1st time looking at a Homebrew formula, no idea how it works)

For our deb/rpm distribution, we (InfluxDB team) decided to create a brand-new package `influxdb2` instead of pushing the 2.x binaries into the existing `influxdb`. The Homebrew parallel in my head would be to create something like `influxdb@2`. WDYT?

Reply:
We're more like Arch Linux in that we just always ship the latest version

Reply:
For more context, we did (accidentally) release 2.x binaries under the `influxdb` package when 2.0 was released. We got some pretty strong negative feedback from users who blindly upgraded, because:
1. 2.x requires auth, so existing unauthed 1.x clients started to fail with 401s / 403s
2. The on-disk storage layout of 2.x is incompatible with 1.x, so after users resolved their auth problems it looked like all of their data had disappeared

`influxd` 2.x includes an `upgrade` subcommand that handles converting 1.x metadata into the 2.x format, as well as rearranging the raw 1.x data into the new file system layout. We chose not to run the upgrade automatically on the deb/rpm side, and instead print a [notice](https://github.com/influxdata/influxdb/blob/master/scripts/post-install.sh#L48-L66) describing the manual steps required.

All that is to say, I'd be careful about treating this as a simple drop-in replacement :smile: If it's possible to automate running `influxd upgrade` after the binary is built, that could be a nice win.

Reply:
It's something that people can/will expect from homebrew.